wgetpaste

```diff
@@ -36,6 +36,10 @@ URL_gists="https://api.github.com/gists"
 ENGINE_tinyurl=tinyurl
 URL_tinyurl="http://tinyurl.com/ api-create.php"
 REGEX_RAW_tinyurl='s|^\(http://[^/]*/\)\([[:alnum:]]*\)$|\1\2|'
+# snippets
+ENGINE_snippets=snippets
+URL_snippets="https://gitlab.com/api/v4/snippets"
+SECOND_HEADER_snippets="Content-Type: application/json"
```

Comment:
Perhaps we should just make the headers a list/array so we don't end up with `THIRD_HEADER` and so on down the line?

Reply:
Sure, will do in a min.

Reply:
In order to preserve any previous "HEADER_" setting, created an array only for any additional header.